lib/ui/home/home_screen.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:survey_flutter_ic/ui/home/home_header.dart';
+
+class HomeScreen extends StatelessWidget {
+  const HomeScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return const AnnotatedRegion<SystemUiOverlayStyle>(
```

Comment:
TODO. Handle `AnnotatedRegion` in `MyApp` instead

Reply:
@Wadeewee Will you update this?

Reply:
Yep @hoangnguyen92dn Please allow me update my the end of the day 🙇‍♀️